helpers/k8s_api_test.go

```diff
@@ -14,12 +14,20 @@ import (
 var _ = Describe("Synthetic Test Assumed role", func() {
 	Context("when calling k8s api with apps namespace and pods kind", func() {
 		It("returns pods list and can access the first image value", func(ctx SpecContext) {
-			podList, _ := k8s_api.GetPodList(ctx, os.Getenv("ENVIRONMENT_NAME"), "apps")
+			aws_account_id, err := k8s_api.GetAwsAccountID(ctx)
+			if err != nil {
```

Comment:
Wouldn't you want the test to fail here, since it couldn't get the account id? I don't think `GinkgoWriter.Printf` causes a test failure.